### PR TITLE
Testing automated rollbacks using shared library.

### PR DESCRIPTION
**Pull request by edgibbs**

### CODE REVIEW

**Comment by edgibbs** [comment]
retest this please

**Comment by edgibbs** [comment]
retest this please

**Comment by edgibbs** [comment]
retest this please

File: Jenkinsfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import groovy.transform.Field
 
-@Library('jenkins-pipeline-utils') _
+@Library('jenkins-pipeline-utils@FIT-206-automatic-rollback') _
```
Comment (money) [inline Jenkinsfile]: Branch needs to be specified for this feature?

File: Jenkinsfile [hunk 2]
```diff
@@ -153,6 +153,7 @@ def smokeTestStage(environment) {
           string(credentialsId: 'dashboard-smoke-password', variable: 'SMOKE_TEST_PASSWORD'),
           string(credentialsId: 'dashboard-smoke-verification-code', variable: 'SMOKE_VERIFICATION_CODE')
         ]) {
+          error('badness in integration')
```
Comment (money) [inline Jenkinsfile]: Don't fully understand, but intentionally raising error because why?

Comment (edgibbs) [reply Jenkinsfile]: oh sorry I should have made this a draft, it was to intentionally fail in a step so that I could test the rollback code.

**Pull request closed without merging**